api/src/pcapi/routes/backoffice/templates/components/links.html

```diff
@@ -60,6 +60,11 @@
      class="link-primary"
      title="Page de l'offre">{{ offer.id }}</a>
 {% endmacro %}
+{% macro build_collective_offer_template_details_link(offer) %}
+  <a href="{{ url_for('backoffice_web.collective_offer_template.list_collective_offer_templates') }}" {# !TODO: need to create function in blueprint in PC-26057 #}
```

Comment:
Ça doit quand même afficher la recherche avec l'id. As-tu testé en local ?
```suggestion
  <a href="{{ url_for('backoffice_web.collective_offer_template.list_collective_offer_templates', q=offer.id) }}" {# !TODO: need to create function in blueprint in PC-26057 #}
```